public/javascripts/i18n.js

```
@@ -0,0 +1,1080 @@
+        var args = slice.call(arguments);
+        var options = {};
+        var subject;
+        while (args.length) {
+            subject = args.shift();
+
+            if (typeof (subject) !== 'object') {
+                continue;
+            }
+
+            for (var attr in subject) {
+                if (!subject.hasOwnProperty(attr)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)